docs/book/15-Exceptions.md

```diff
@@ -15,7 +15,7 @@ Java 使用异常来提供一致的错误报告模型，使得构件能够与客
 
 Java 中的异常处理的目的在于通过使用少于目前数量的代码来简化大型、可靠的程序的生成，并且通过这种方式可以使你更加确信：你的应用中没有未处理的错误。异常的相关知识学起来并非艰涩难懂，并且它属于那种可以使你的项目受益明显、立竿见影的特性之一。
 
-因为异常处理是 Java 中唯一官方的错误报告机制，并且通过编译器强制执行，所以不学习异常处理的话，书中也就只能写出那么些例子了。本章将向读者介绍如何编写正确的异常处理程序，并将展示当方法出问题的时候，如何产生自定义的异常。
+因为异常处理是Java报告错误的唯一官方方式，并被编译器强制要求，如果不学习异常处理的话，Java学习不会很深入。本章将教你如何编写正确的异常处理程序，以及当方法出问题的时候，如何产生自定义的异常。
```

Comment:
改了之后很奇怪。贴原文

Reply:
原翻译更奇怪。
Because exception handling is the only official way that Java reports errors, and is enforced by the Java compiler, you can only go so far without learning about exception handling. This chapter teaches you to write code to properly handle exceptions, and to generate your own exceptions if one of your methods gets into trouble.

Reply:
and is enforced by the Java compiler

这个原翻译好

Reply:
you can only go so far without learning about exception handling

不学习异常处理的话，Java 学习也就到此为止

Reply:
> and is enforced by the Java compiler
> 
> 这个原翻译好

原翻译：“并且通过编译器强制执行”有歧义，会被误以为是编译器强制执行异常处理。原意是编译器强制要求程序员进行异常处理。

Reply:
> you can only go so far without learning about exception handling
> 
> 不学习异常处理的话，Java 学习也就到此为止



> you can only go so far without learning about exception handling
> 
> 不学习异常处理的话，Java 学习也就到此为止

都可以，我翻译的是意译，这句话直译很奇怪。“不学习异常处理的话，Java 学习也就到此为止”有歧义，不学异常处理，Java学习就完成了。

Reply:
> > and is enforced by the Java compiler
> > 这个原翻译好
> 
> 原翻译：“并且通过编译器强制执行”有歧义，会被误以为是编译器强制执行异常处理。原意是编译器强制要求程序员进行异常处理。

强制要求也不太好，问题都出现在了隐含一次没有完整表述。要么就把隐含一次都补出来

Reply:
我直接改回原翻译